CONTRIBUTING.md

```diff
@@ -2,8 +2,8 @@
 
 > Any change matters, whatever the size, just do it.
 
-We are happy that you are willing to contribute. For that reason, we
-have prepared these guidelines for you:
+We're happy that you're considering contributing! To help, we've prepared these
```

Comment:
I'd prefer don't use abbreviation on the documentation, looks a bit informal. Could we restore them?

https://www.grammarly.com/blog/abbreviations/

> One thing to remember about abbreviations is that certain ones are considered informal. If you are writing something very formal, it’s better to err on the side of spelling things out. The other thing to remember is that some readers may not know what an abbreviation means. If the abbreviation is obscure or unfamiliar, make sure to explain what it means the first time you use it.

Example: https://yarnpkg.com/advanced/contributing

Reply:
@juanpicado You're quoting text that suggests this rule applies when "writing something very formal" and you've linked as an example to a piece of text that uses a very similar abbreviation 3 sentences in...?

Reply:
Yeah, I bad example from my side. My point is to be consistent with the current documentation, eg: https://github.com/verdaccio/website/blob/master/docs/best-practices.md

Eventually those files will be hosted also in this repo. I'm not a technical writer, so I'm not giving advice in such level, maybe you are I don't know. 

I was just suggesting consistency. Somehow I appreciate you improve the reading 👍🏼